app/util/AWS.scala

```diff
@@ -24,6 +25,15 @@ class AWSConfig @Inject() (config: Configuration) {
 
   lazy val credProvider = new AWSCredentialsProviderChain(credProviders.flatten: _*)
 
+  lazy val sessionId: String = "session" + Math.random()
+
+  lazy val kinesisArn = config.getString("aws.kinesis.streamArn").getOrElse("")
+
+  lazy val atomsCredProvider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("composer"),
```

Comment:
The code and prod kinesis streams are in the composer account. Having the composer credentials provider here should allow people to access the code/prod streams locally.